tests/playground.html

```diff
@@ -228,8 +228,8 @@
           blockToDrag.onMouseUp_(mouseUpEvent);
 
           setTimeout(fakeDragWrapper(), 100);
-        }, 30);
-    }, 30);
+        }, 100);
```

Comment:
is this intentionally part of this PR? Fine if it is, just making sure you meant for it to be here.